docs/usage.md

```diff
@@ -903,52 +880,106 @@ snpeff_db         = '105'
+    --snpeff_cache /data/cache/snpeff/ \
+    --vep_cache /data/cache/ensemblvep/ \
     ...
 ```
 
-Or similarly on the cloud:
+Alternatively the data may be stored on AWS S3 storage, therefore the parameters might be:
 
 ```bash
-s3://data/igenomes/
-s3://data/cache/ensemblvep
-s3://data/cache/snpeff
+s3://my-reference-data/igenomes/
+s3://my-reference-data/cache/ensemblvep/
+s3://my-reference-data/cache/snpeff/
 ```
 
 which can then be used this way in sarek:
 
 ```bash
-nextflow run nf-core/sarek \\
-    --igenomes_base s3://data/igenomes/ \\
-    --snpeff_cache s3://data/cache/snpeff/ \\
-    --vep_cache s3://data/cache/ensemblvep/ \\
+nextflow run nf-core/sarek \
+    --igenomes_base s3://my-reference-data/igenomes/ \
+    --snpeff_cache s3://my-reference-data/cache/ensemblvep/ \
+    --vep_cache s3://my-reference-data/cache/snpeff/ \
     ...
 ```
 
 These params can be specified in a config file or in a profile using the params scope, or even in a json or a yaml file using the `-params-file` nextflow option.
 
+### Use pre-populated cache for SNPeff and VEP
+
+Sarek stores a mirror of some cache files on AWS S3 which are set as the default parameters. They contain some genome builds which can be found by checking the contents of the S3 bucket.
+
+```bash
+vep_cache = s3://annotation-cache/vep_cache/
+```
+
+The contents of the VEP cache can be listed with the following command using the S3 CLI:
+
+```bash
+aws s3 --no-sign-request ls s3://annotation-cache/vep_cache/
+```
+
+Simialrly, the SNPeff cache is stored at the following location on S3:
+
+```bash
+snpeff_cache = s3://annotation-cache/snpeff_cache/
+```
+
+The contents can be listed as follows:
+
+```bash
+aws s3 --no-sign-request ls s3://annotation-cache/snpeff_cache
+```
+
+Be sure the specified species, genome and build matches the directory structure within the cache!
+
+### Use Sarek to download cache and annotate in one go
+
+Use the params `--download_cache`, and specify with `--tools` for which annotation tool you need to download the cache (`snpeff` and or `vep`)
+
+Sarek will automatically download the cache, use the biocontainers container for said tools, and use it to annotate any vcfs produced.
+
+### Only download cache
+
+Using the params `--build_only_index` allow for only downloading the cache for the specified tools.
+
+### Location for the cache
+
+Cache can be downloaded in the specified `--outdir_cache` location.
+Else, it will be downloaded in `cache/` in the specified `--outdir` location.
+
+To download cache on a cloud infrastructure, an absolute path is needed.
+
+### Using the nf-core containers with pre-downloaded cache
+
+For common genomes, the VEP and snpEff parameters are pre-populated in the [igenomes.config](https://github.com/nf-core/sarek/blob/master/conf/igenomes.config) file and as such the cache files are included with the specified iGenome. However, these are infrequently updated and therefore using a pre-populated cache is preferable.
```

Comment:
```suggestion
For common genomes, the VEP and snpEff parameters are pre-populated in the [igenomes.config](https://github.com/nf-core/sarek/blob/master/conf/igenomes.config) file and as such the cache files are included with the specified iGenome. However, these are infrequently updated and therefore using the newest available cache is preferable.
```
or sth like that?
I understood it as not using the pre-populated cache is preferred to get the newest of the newest?